### DISCUSSION TITLE
Using the App

### DISCUSSION THREAD

**Discussion by Jmidd56** [mrkai77/Loop#329] [Q&A]
How do you actually use the app? I just downloaded it because it was being promoted in the MacApps discord and I am a little confused.

**Comment by mrkai77** [comment]
No problem!

First, you will need to open the app (right click -> open may be necessary), and then you will need to grant accessibility permissions to the app. After that, you can focus any other app and trigger Loop's radial menu using the <kbd>Globe</kbd> key (this can be configured in Loop's settings). With the radial menu open, if you move your mouse to a specific direction, you can move windows towards that direction :)

Please let me know if you are confused about anything else :)
